esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -677,21 +779,43 @@ def __init__(
             exceeds ``mdtol``. ``mdtol=0`` means no missing data is tolerated while
             ``mdtol=1`` will mean the resulting element will be masked if and only
             if all the contributing elements of data are masked.
+        method : str, default="conservative"
+            The regridding method which :mod:`ESMF` uses to calculate weights. Can be
+            either "conservative" or "bilinear".
 
         """
+        if method not in ["conservative", "bilinear"]:
+            raise (ValueError("method must be either bilinear or conservative."))
```

Comment:
Use a `NotImplementedError`, referencing the invalid argument that was provided.